### PR TITLE
CHIME/FRB Mission Page

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
# Description

PR for adding CHIME/FRB mission page over GCN website.

# Related Issues

Please find how content looks like at website:
Screenshot 2024-11-12 at 2 44 46 PM

### CODE REVIEW

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please resolve merge conflicts.

**Comment by Vidushi-GitHub** [comment]
https://github.com/tabbott36 Please provide your feedback.

File: app/routes/missions.chime/route.mdx [hunk 1]
```diff
@@ -5,9 +5,66 @@ handle:
...
 
-# Chime
+# Canadian Hydrogen Intensity Mapping Experiment (CHIME)
+
+<img
+  src={logo}
+  align="right"
+  width="568"
+  height="420"
+  alt="Einstein Probe logo"
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:22]: Copy-paste error.

File: app/routes/missions.chime/route.mdx [hunk 2]
```diff
@@ -5,9 +5,66 @@ handle:
...
 }
 
-# Chime
+# Canadian Hydrogen Intensity Mapping Experiment (CHIME)
+
+<img
+  src={logo}
+  align="right"
+  width="568"
+  height="420"
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:21]: These dimensions don't match the aspect ratio of the image.

File: app/routes/missions.chime/route.mdx [hunk 3]
```diff
@@ -5,9 +5,66 @@ handle:
...
+  alt="Einstein Probe logo"
+  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
+/>
+
+**Launch Date:** September 7, 2017
+
+The [Canadian Hydrogen Intensity Mapping Experiment (CHIME)](https://chime-experiment.ca/en) is a transit radio telescope
+located in Penticton BC, Canada. It consists of 4 semi cylindrical reflectors, each with 2000
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:29]: Is that how Canadian cities and provinces are punctuated? If it's like American cities and states, then there should be a comma between the city and the province.

Also, "semicylindrical" is a single word.

```suggestion
located in Penticton, BC, Canada. It consists of 4 semicylindrical reflectors, each with 2000
```

File: app/routes/missions.chime/route.mdx [hunk 4]
```diff
@@ -5,9 +5,66 @@ handle:
...
+square meters of collecting area and 256 dual-polarization antennas. The telescope
+observes between 400 MHz and 800 MHz and covers an instantaneous field of view of ~200
+square degrees. CHIME houses several electronic “backends” which are tailored for specific
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:32]:
```suggestion
square degrees. CHIME houses several electronic backends which are tailored for specific
```

File: app/routes/missions.chime/route.mdx [hunk 5]
```diff
@@ -5,9 +5,66 @@ handle:
...
+scientific goals, such as generating cosmological maps of hydrogen density, detecting short
+duration radio transients known as Fast Radio Bursts (FRBs), and observing and timing
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:34]:
```suggestion
scientific goals, such as generating cosmological maps of hydrogen density, detecting fast radio bursts (FRBs), and observing and timing
```

File: app/routes/missions.chime/route.mdx [hunk 6]
```diff
@@ -5,9 +5,66 @@ handle:
...
+
+| Instruments | Energy Range  | Field of View        | Localization |
+| ----------- | ------------- | -------------------- | ------------ |
+| CHIME/FRB   | 400 - 800 MHz | ~200 deg<sup>2</sup> | ~arcmin      |
+
+</div>
+
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+CHIME distributes alerts for the detection of fast radio bursts. These notices are published on the GCN Kafka topic `gcn.notices.chime.frb.alert`.
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:48]: Why not `gcn.notices.chime.frb`?

File: app/routes/missions.chime/route.mdx [hunk 7]
```diff
@@ -5,9 +5,66 @@ handle:
...
+
+</div>
+
+**Yearly Trigger Rates:**
+
+<div className="overflow-table">
+
+| Instrument | Type              | Rates |
+| ---------- | ----------------- | ----- |
+| CHIME/FRB  | Fast Radio Bursts | ~600  |
```
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:68]: What units? 600 is a dimensionless number, not a rate.

File: app/routes/missions.tsx [hunk 8]
```diff
@@ -35,7 +35,7 @@ export default function () {
               </NavLink>,
               useFeature('CHIME') && (
                 <NavLink key="chime" to="chime">
-                  Chime
+                  CHIME/FRB
```
Comment (lpsinger) [inline app/routes/missions.tsx:38]:
```suggestion
                  CHIME
```

File: app/routes/missions.chime/route.mdx
Comment (lpsinger) [inline app/routes/missions.chime/route.mdx:1]: Update breadcrumb in header to match capitalization: CHIME, not Chime.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Do we have a point of contact from CHIME that we can add as a reviewer?

**Comment by Vidushi-GitHub** [comment]
> Do we have a point of contact from CHIME that we can add as a reviewer?

Kindly add https://github.com/tabbott36 as a reviewer, I am unable to add.

File: app/routes/missions.chime/route.mdx (hunk 7 above)
Comment (Vidushi-GitHub) [reply app/routes/missions.chime/route.mdx:68]: It's yearly trigger rate as per heading.

**Comment by lpsinger** [comment]
> > Do we have a point of contact from CHIME that we can add as a reviewer?
> 
> Kindly add https://github.com/tabbott36 as a reviewer, I am unable to add.

@tabbott36, we sent you an invitation to join our GitHub org. Would you please accept it?

**Comment by tabbott36** [comment]
This looks great to me!

**Review by lpsinger — APPROVED** [review]

**Pull request merged**